### PR TITLE
Eobs 285 print report contains section patient monitoring exceptions

### PR DESCRIPTION
**Pull request by moderatemisbehaviour**
* The **Patient Monitoring Exceptions** section of the observation report contains information about the newer style patient monitoring exceptions between the dates specified.

* Patient monitoring exceptions ending due to transfers are also represented in the **Patient Monitoring Exceptions** section.

* The **Actions Triggered** section contains information about EWS observations that were cancelled due to a patient monitoring exception taking effect.

### CODE REVIEW

File: nh_eobs/views/observation_report_template.xml [hunk 1]
```diff
@@ -813,41 +818,10 @@
...
-                                                                    </table>
-                                                                </td>
-                                                            </tr>
-                                                        </t>
-                                                    </table>
-                                                </td>
-                                            </tr>
+                                            <t t-call="nh_eobs.triggered_actions"/>
+                                        </t>
+                                        <t t-if="'state' in observation and observation['state'] == 'cancelled'">
```
Comment (Gimpneek) [inline nh_eobs/views/observation_report_template.xml]: Will this separation of full obs and partial (cancelled) obs change the chronological order of the events in the report as currently the report reflects all aspects of the patient journey in this order where as this shows the full obs in chronological order and then the partial (cancelled ) obs in chronological order.

Also what does this mean for actual partial (non-cancelled) obs

Comment (moderatemisbehaviour) [inline nh_eobs/views/observation_report_template.xml]: I looked at the template and thought that too, the diff is messing with my head though, I think it's actually fine! Check it out [here](https://github.com/NeovaHealth/openeobs/blob/fede6251edb19b121ccb8c49589072eac53866f0/nh_eobs/views/observation_report_template.xml#L819) instead.

I did a test where I stopped and restarted obs to cause a cancellation, and then did an obs with some other escalation tasks. The cancelled obs appears correctly in chronological order.

**Comment by Gimpneek** [comment]
:shipit: Once added a few more unit tests around cancel_reason kwarg added to methods

**Comment by Nayira** [comment]
Test cases for EOBS-285, EOBS-454, EOBS-455 and EOBS-501 have Passed 
:shipit: :shipit: :shipit: :shipit: :shipit: :shipit: !!!

**Pull request merged**